libs/blocks/global-navigation/global-navigation.js

```diff
@@ -213,6 +213,10 @@ class Gnav {
   ims = async () => loadIms()
     .then(() => this.imsReady())
     .catch((e) => {
+      if (e?.message === 'IMS timeout') {
```

Comment:
Is it always exactly this message? No suffix/prefix/other capitalisation/etc.?

Reply:
Yup, we throw it ourselves https://github.com/adobecom/milo/blob/main/libs/utils/utils.js#L673 so if that changes we either consider it.. or see it popping up in the lana logs again

Reply:
Oh, right, I thought it's coming from IMS